portfolio/src/main/webapp/script.js

```diff
@@ -75,9 +75,29 @@ function addRandomFact(){
     if(facts.length == 0){
         factContainer.innerText = "Thats all the facts about me !";
     } else {
-    // Pop element via index. 
-    facts.splice(idx,1);
-    // Add it to the page.
-    factContainer.innerText = fact;
+        // Pop element via index. 
+        facts.splice(idx,1);
+        // Add it to the page.
+        factContainer.innerText = fact;
     }
 }
+
+function introduce(){
+    fetch('/data').then(response => response.text()).then((fact) => {
+    document.getElementById('fact-container').innerText = fact;
+  })};
+
+function fetchData(){
+    fetch('/data')
+    .then(response => response.json())
```

Comment:
is this method just for debugging?

Reply:
I wanted to print the json to the console i was confused about how to do that though.